src/components/accessories/admin/operations/operationTable/OperationTable.tsx

```diff
@@ -122,6 +133,26 @@ export const OperationTable: FunctionComponent<IOwnProps> = ({
                       type: operation.type?.code,
                     })) ?? []
                   }
+                  headerActions={headerActions}
+                />
+                {deleteOperation.status === "FAIL" && (
+                  <div ref={infoBoxRef} className="info-box-container">
+                    <InfoBox
+                      type="error"
+                      message={deleteOperation.error?.message}
+                    />
+                  </div>
+                )}
+                <ConfirmationDialog
+                  isOpen={!!deleteOperation.hasSucceeded}
```

Comment:
Just out of curiosity, what is the difference between `.hasSucceded` and `.status === "SUCCESS"`
IMO, using only `.status` would be more consistent and clearer for other developers. wdyt?

![Screen Recording 2024-05-30 at 15 00 32](https://github.com/informatici/openhospital-ui/assets/597067/d4896aa7-cb5a-4695-85da-bfe5f7abd4cb)

```suggestion
                  isOpen={deleteOperation.status === "SUCCESS"}
```